docker-compose.circleci.yml

```diff
@@ -23,6 +23,12 @@ services:
       - 11001:8080
     networks:
       - circleci-prisoner_content_hub
+  chrome:
+    image: previousnext/chrome-headless:65
```

Comment:
bit of a random org to be installing docker images from. Have you tried any of the circle images? https://circleci.com/developer/images/image/cimg/node#browsers

These tend to be much faster as massively cached but also come from a trusted source

Reply:
or maybe https://circleci.com/developer/orbs/orb/circleci/browser-tools?

Reply:
This came from the Drupal test traits library that we're using.  https://gitlab.com/weitzman/drupal-test-traits/-/blob/master/docker-compose.yml#L47
I didn't try out any other images (or work out what this specific image is doing differently).  It might be that the Drupal test traits library has a dependency on that particular image (if this is the case then I guess we should document it).
I could take a further look in another PR?